statsmodels/tsa/statespace/tests/test_initialization.py

```diff
@@ -374,34 +369,54 @@ def test_nested():
 
 def test_invalid():
     # Invalid initializations (also tests for some invalid calls to set)
-    assert_raises(ValueError, Initialization, 5, '')
-    assert_raises(ValueError, Initialization, 5, 'stationary', constant=[1, 2])
-    assert_raises(ValueError, Initialization, 5, 'stationary',
-                  stationary_cov=[1, 2])
-    assert_raises(ValueError, Initialization, 5, 'known')
-    assert_raises(ValueError, Initialization, 5, 'known', constant=[1])
-    assert_raises(ValueError, Initialization, 5, 'known', stationary_cov=[0])
+    with pytest.raises(ValueError):
+        Initialization(5, '')
```

Comment:
I don't like pytest for simple assert_raises
but here it is Chad's code.

Reply:
The contextmanager version is much friendlier for copy/pasting when running in the REPL, plus readability.

> but here it is Chad's code.

Yep, I can point you towards some more trivial PRs in non-Chad's code if that would help.